mobile/src/bundles/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -2,27 +2,122 @@ import React from 'react';
 
 import {
     Button,
-    Link,
+    FormField,
+    Input,
     Text,
+    TouchableOpacity,
+    useLinkTo,
     View,
 } from '~/bundles/common/components/components';
-import { RootScreenName } from '~/bundles/common/enums/enums';
+import { RootScreenName, TextCategory } from '~/bundles/common/enums/enums';
+import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/global-styles';
+
+import { USER_SIGN_IN_DEFAULT_VALUES } from './constants/constants';
+import { styles } from './styles';
 
 type Properties = {
     onSubmit: () => void;
 };
 
-const SignInForm: React.FC<Properties> = () => {
+const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
+    const { control, errors, handleSubmit } = useAppForm({
+        defaultValues: USER_SIGN_IN_DEFAULT_VALUES,
+    });
+
+    const linkTo = useLinkTo();
+
+    const handleFormSubmit = useCallback((): void => {
+        void handleSubmit(onSubmit)();
+    }, [handleSubmit, onSubmit]);
+
     return (
-        <View>
-            <Text>Sign In</Text>
-            <Button
-                label="Sign in"
-                onPress={(): void => {
-                    // TODO: handle Sign in
-                }}
-            />
-            <Link label="Go to Sign Up" to={`/${RootScreenName.SIGN_UP}`} />
+        <View
+            style={[
+                globalStyles.ph25,
+                globalStyles.flex1,
+                globalStyles.justifyContentCenter,
+            ]}
+        >
+            <Text
+                category={TextCategory.H4}
+                style={[
+                    globalStyles.pb25,
+                    globalStyles.alignSelfCenter,
+                    styles.title,
+                ]}
+            >
+                Hi! Login to your Account
+            </Text>
+            <View style={styles.formWrapper}>
+                <FormField errors={errors} label="Email" name="email" required>
```

Comment:
```suggestion
                <FormField errors={errors} label="Email" name="email">
```
On the design we don't have required